geode-redis/src/main/java/org/apache/geode/redis/internal/RedisStats.java

```diff
@@ -82,6 +104,28 @@
     expirationTimeId = type.nameToId("expirationTime");
   }
 
+  public RedisStats(StatisticsFactory factory, StatisticsClock clock) {
+    this(factory, "redisStats", clock);
+  }
+
+  public RedisStats(StatisticsFactory factory, String textId, StatisticsClock clock) {
+    stats = factory == null ? null : factory.createAtomicStatistics(type, textId);
+    this.clock = clock;
+    this.START_TIME_IN_NANOS = this.clock.getTime();
+    perSecondExecutor = startPerSecondUpdater();
+  }
+
+  public void clearAllStats() {
```

Comment:
For consistency, I think it would be good to reset the other (Geode-specific) stats here too (not just `clientId`).

Reply:
agree, thanks.

Reply:
Not necessary to do this in this PR unless you're making other changes as well, but it might be a good idea to make this method `protected` instead of public and mark it as `@VisibleForTesting`